crates/astria-composer/src/executor/mod.rs

```diff
@@ -276,7 +287,122 @@ impl Executor {
                     }
                 }
             }
+        };
+
+        self.status
+            .send_modify(|status| status.is_connected = false);
+
+        // close the channel to avoid receiving any other txs before we drain the remaining
+        // sequence actions
+        self.serialized_rollup_transactions.close();
+
+        match &reason {
+            Ok(reason) => {
+                info!(reason, "shutting down");
+            }
+            Err(reason) => {
+                error!(%reason, "executor exited with error");
+                // we error out because of a failure to submit a bundle to the sequencer
+                // we do not want to proceed with the shutdown process in this case
+                return Err(eyre!(reason.to_string()));
+            }
+        };
+
+        let mut bundles_to_drain: VecDeque<SizedBundle> = VecDeque::new();
+        let mut bundles_drained = 0;
+
+        info!("draining sequence actions from the executor receiver channel");
```

Comment:
All of these words are implementation details that you don't need in logs.
```suggestion
        info!("draining already received transactions");
```